ncm-ceph/src/main/pan/components/ceph/schema-rgw.pan

```diff
@@ -0,0 +1,30 @@
+declaration template components/${project.artifactId}/schema-rgw;
+
+@documentation{ ceph rados gateway config }
+type ceph_radosgw_config = { 
+    include ceph_daemon_config
+    'host'      : string
+    'keyring'   : string
+    'rgw_socket_path' : string = ''
+    'log_file'  : string = '/var/log/radosgw/client.radosgw.gateway.log'
+    'rgw_frontends' : string = "\"civetweb port=8000\""
```

Comment:
It is currently unclear to me what it will actually parse properly. Judging by [this bug report](https://bugzilla.redhat.com/show_bug.cgi?id=1228845), it seems that leaving the quotes in is the safest bet.
Ken Dreyer, in comment 3 on that is referring to [this PR](https://github.com/ceph/ceph/pull/5103), which suggests that even the Ceph developers are not quite sure what it can and should parse properly.
I suggest we leave the quotes, watch that PR and revisit this when they've resolved it on the Ceph side.